Enterprise-CMCS/macpro-onemac
# Update Formal RAI Response header/field name for CMS users

Story: https://qmacbis.atlassian.net/browse/OY2-24636

### Details

This PR changes the field/column label of "Formal RAI Received" to "Formal RAI Response" for CMS users, aligning it with what State users already see the title as.

### Changes

- Field name changed; no more conditional for CMS users
- Column name changed

### Implementation Notes

- N/A

### Test Plan

1. Login as a CMS user
2. Ensure the column name reflects the AC
3. Ensure the field name reflects the AC
4. Log in as a State user
5. Ensure the column and field names have not changed
